### ISSUE TITLE
stubPath .... is not a valid directory

### ISSUE THREAD

**Issue by orloff34** [microsoft/pylance-release#1075]
I am new with python and VS Code. I am using VS Code 1.54.3 with Ubuntu 20.04 and Pylance 2021.3.2 and I am getting following output at start-up of VS Code:

> Info  - 09:16:30] Pylance language server 2021.3.2 (pyright 85309906) starting
[Info  - 09:16:30] Server root directory: /home/xxx/.vscode/extensions/ms-python.vscode-pylance-2021.3.2/dist
[Info  - 09:16:30] Background analysis(3) root directory: /home/xxx/.vscode/extensions/ms-python.vscode-pylance-2021.3.2/dist
[Info  - 09:16:30] Background analysis(3) started
[Info  - 09:16:30] Background analysis(4) root directory: /home/xxx/.vscode/extensions/ms-python.vscode-pylance-2021.3.2/dist
[Error - 09:16:30] stubPath /home/xxx/Nextcloud/Python/VSCodeProjects/Milestone-2/typings is not a valid directory.
[Error - 09:16:30] stubPath /home/xxx/Nextcloud/Python/VSCodeProjects/import_project/typings is not a valid directory.
[Error - 09:16:31] stubPath /home/xxx/Nextcloud/Python/VSCodeProjects/milestone_2_lists/typings is not a valid directory.
[Error - 09:16:31] stubPath /home/xxx/Nextcloud/Python/VSCodeProjects/Database/typings is not a valid directory.

Can you explain why this is displayed and how to solve the problem.

**Comment by jakebailey** [comment]
This isn't a real error; you don't need a typings directory to work so you can ignore it.

I think "error" is a bit too strong for this message; perhaps we can just change it to a warning.

**Comment by erictraut** [comment]
I like the suggestion of changing this from an error to a warning. That should help reduce confusion about this message. I just made that change.

**Comment by jakebailey** [comment]
Thanks, that's all we need. 🙂

**Comment by jakebailey** [comment]
This issue has been fixed in version 2021.3.3, which we've just released. You can find the changelog here: https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#202133-24-march-2021

**Comment by ssbarnea** [comment]
@jakebailey That bug should have not being closed because even a warning is confusing, especially when there is nothing the user can to get this sorted.

Setting the settings to empty string does not silence this message. If the user changes the vscode user settings, they will get an additional message `The stubPath has been specified in both the config file and the client settings. The value in the config file (...) will take precedence`.

That drives me crazy as even setting it to another existing folder does not silence the language server. I tried .tox and I got `stubPath .tox is not a valid directory.`.
